portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -35,38 +46,60 @@ public void init(){
 
     @Override
     public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {  
+        // Send a query for the comments, latest first
+        Query query = new Query("comments").addSort("timestamp", SortDirection.DESCENDING);
```

Comment:
Line is longer than 80 characters. Please format to be under 80 characters.

Reply:
(pulling from convo on Ibiyemi's PR) it looks like Java should be under 100 characters (https://google.github.io/styleguide/javaguide.html#s4.4-column-limit)